models/intel/gaze-estimation-adas-0002/description/gaze-estimation-adas-0002.md

```diff
@@ -17,62 +16,54 @@ The network takes three inputs: square crop of left eye image, square crop of ri
 |-----------------------|---------------------------------------------|
 | GFlops                | 0.139                                       |
 | MParams               | 1.882                                       |
-| Source framework      | Caffe2                                      |
-
+| Source framework      | Caffe2\*                                    |
```

Comment:
is it really caffe2? I believe it is PyTorch

Reply:
It will be fixed in separate request